apps/dashboard/src/main/java/com/akto/action/testing/StartTestAction.java

```diff
@@ -582,13 +583,21 @@ public String fetchTestingRunResults() {
         return SUCCESS.toUpperCase();
     }
 
+    List<TestingRunIssues> allIgnoredTestingRunIssues;
     public String fetchVulnerableTestRunResults() {
         ObjectId testingRunResultSummaryId;
         try {
             testingRunResultSummaryId = new ObjectId(testingRunResultSummaryHexId);
+            allIgnoredTestingRunIssues = TestingRunIssuesDao.instance.findAll(Filters.and(
+                    Filters.eq(TestingRunIssues.LATEST_TESTING_RUN_SUMMARY_ID, testingRunResultSummaryId),
```

Comment:
remove from here, this is used with skip{ so it will be called everytime, we dont need that